db/migrate/20181010134649_add_evm_owner_to_orchestration_stacks.rb

```diff
@@ -1,7 +1,12 @@
 class AddEvmOwnerToOrchestrationStacks < ActiveRecord::Migration[5.0]
+  class OrchestrationStack < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
   def change
     add_reference :orchestration_stacks, :evm_owner, :type => :bigint, :index => true
     add_reference :orchestration_stacks, :miq_group, :type => :bigint, :index => true
     add_reference :orchestration_stacks, :tenant,    :type => :bigint, :index => true
+    OrchestrationStack.reset_column_information
```

Comment:
This implies that a *previous* migration did not use an OrchestrationStack stub properly, I think.

Reply:
To catch up with what our discussion was in gitter yesterday:

1) Using a stub at all causes the table schema/columns for that stub to be cached
2) If subsequent migration changed the schema of this table...
3) and another migration after this schema change tries to use a stub for this table, it would get cached information

It's unclear which migration author would know to reset column information other than the maybe the first two:
1) after you use any stub, you need to reset column information, YUCK
2) if you change the schema of a table, you should add a stub for the model and reset column information of that stub after you changed the schema (less yuck)
3) This migration is a victim... it seems odd to reset column information before using any stub... we can but YUCK